ui/src/app/dashboard/dashboard.resolver.service.ts

```diff
@@ -12,7 +14,7 @@ export class DashboardResolver implements Resolve<AggregationResponse>{
   ) {}
 
   resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Promise<AggregationResponse> {
-    return this.jobManagerService.queryAggregations(TimeFrame.DAYS7,
+    return this.jobManagerService.queryAggregations(stringToTimeFrameMap.get(route.queryParams['timeFrame']),
```

Comment:
Doesn't this need to handle the default case as well? What if the query param is not given?

Reply:
In default case we also encode the timeframe in the URL, we want to redirect user if the timeframe is not in the url. 
Adding a TODO here to reminder the "redirect" thing.